### PR TITLE
Use Elastic-Git for Git access in the Django Grapelli CMS

### PR DESCRIPTION
**Pull request by smn**
This is the CMS side of https://github.com/praekelt/unicore-cms/pull/36

### CODE REVIEW

File: cms/tests/test_tasks.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            index_prefix='%s_remote' % (self.local_workspace.index_prefix,))
+
+        for data, count in self.create_page_data_iter():
+            data['title'] = 'remote page %i' % (count,)
+            self.remote_workspace.save(
+                Page(data), 'Saving %(title)s.' % data)
+        self.remote_workspace.refresh_index()
+
+    def test_push_to_git(self):
+        # NOTE: In order to push to the remote workspace's master branch it
```
Comment (miltontony) [inline cms/tests/test_tasks.py:23]: @smn I take it my multiple remotes fix should allow you to do this better?

Comment (smn) [inline cms/tests/test_tasks.py:23]: No, this is about making sure the branch we're pushing to isn't the branch that is currently checked out in the repository.

**Comment by smn** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
:+1: :100: :+1: :100: :+1: :100: :+1: :100:

**Pull request merged**